api/v2/views/instance.py

```diff
@@ -44,6 +54,7 @@ class InstanceViewSet(MultipleFieldLookup, AuthModelViewSet):
 
     queryset = Instance.objects.all()
     serializer_class = InstanceSerializer
+    filter_class = InstanceFilter
```

Comment:
Great example of how to add Filtering to _any_ API Viewset!

Reply:
👍